operators/src/engine/execution_context.rs

```diff
@@ -131,6 +131,7 @@ impl MockExecutionContext {
         MockQueryContext {
             chunk_byte_size,
             thread_pool: self.thread_pool.clone(),
+            extensions: Default::default(),
             abort_registration,
             abort_trigger: Some(abort_trigger),
         }
```

Comment:
I get the following errors all the time:

> OpenTelemetry trace error occurred. Exporter jaeger encountered the following error(s): thrift agent failed with not open

Yes, I don't have Jeager running. But there shouldn't be this error, right?

Reply:
You should disable the opentelemetry tracing in the Settings.toml if you don't want to use it.